CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [7.0.2]
+### Fixed
+- `createNewVaultAndRestore` now accepts a seedphrase argument passed as an arrays of numbers ([#138](https://github.com/MetaMask/KeyringController/pull/138))
```

Comment:
Nit: Slight re-wording, since it's only one array of numbers and the context was singular ("A seedphrase").

```suggestion
- `createNewVaultAndRestore` now accepts a seedphrase formatted as an array of numbers ([#138](https://github.com/MetaMask/KeyringController/pull/138))
```